### ISSUE TITLE
Select component lacking keyboard accessibility

### ISSUE THREAD

**Issue by mtleonard120** [searchspring/snap#526]
The `Select` component in the snap-preact-components package is inaccessible for keyboard users when not opting to use the `native` prop. At a minimum the non-native select should support keyboard controls for tabbing, opening/closing, and changing selection.

**Comment by korgon** [comment]
Thanks @mtleonard120 - I've added this to our queue.
